pkg/evaluator/evaluator.go

```diff
@@ -126,6 +135,36 @@ func (e *Evaluator) evalReturn(scope *scope, ret *parser.Return) Value {
 	return &ReturnValue{Val: val}
 }
 
+func (e *Evaluator) evalIf(scope *scope, i *parser.If) Value {
+	val := e.evalCoditionedBlock(scope, i.IfBlock)
+	if val == TRUE || isError(val) {
+		return val
+	}
+	for _, elseif := range i.ElseIfBlocks {
+		val := e.evalCoditionedBlock(scope, elseif)
+		if val == TRUE || isError(val) {
+			return val
+		}
+	}
+	if i.Else != nil {
+		return e.Eval(newInnerScope(scope), i.Else)
+	}
+	return nil
```

Comment:
This should be `return FALSE` to mirror the case when a branch is executed and `TRUE` is returned. Or the `TRUE` cases should return `nil` too.

Reply:
ok, done.